presto-hive/src/main/java/com/facebook/presto/hive/HiveSessionProperties.java

```diff
@@ -101,6 +106,16 @@ public HiveSessionProperties(HiveClientConfig config)
                         "Experimental: ORC: Validate writer files",
                         config.isOrcWriterValidate(),
                         false),
+                stringSessionProperty(
```

Comment:
Why are these hidden and experimental? I don't see any reason why we would change or remove these.